packages/@jsii/java-runtime/.gitignore

```diff
@@ -1,3 +1,4 @@
+.m2
```

Comment:
Why oh why not in a tempdir somewhere?

Reply:
Because local iterations will be *much* faster for not re-downloading things every single time. You can nuke this directory when you need it...